aequilibrae/transit/gtfs_loader.py

```diff
@@ -234,13 +232,9 @@ def __load_fare_data(self):
             if fr.route in self.routes:
                 fr.route_id = self.routes[fr.route].route_id
             fr.agency_id = self.agency.agency_id
-            for x in [fr.origin, fr.destination]:
-                if x not in corresp:
-                    corresp[x] = zone_id
-                    zone_id += 1
-            fr.origin_id = corresp[fr.origin]
-            fr.destination_id = corresp[fr.destination] if fr.destination == "" else fr.destination_id
-            self.fare_rules.append(fr) if fr.origin == "" else fr.origin_id
+            fr.origin_id = int(fr.origin) if len(fr.origin) > 0 else None
+            fr.destination_id = int(fr.destination) if len(fr.destination) > 0 else None
```

Comment:
Are you sure the ID has to be numeric, @r-akemii ?

Reply:
I just checked the GTFS documentation and you're right! I used the wrong data type for origin/destination ID. It should be a sequence of UTF-8 characters. I'll fix it soon.

![image](https://github.com/user-attachments/assets/7664c851-55f8-44d8-9fa5-92d80ba241da)

![image](https://github.com/user-attachments/assets/ea841625-69b3-49fd-ac01-898927817270)